<?xml version="1.0"?>
<launch>

  <node pkg="joy" type="joy_node" name="joy_node" />

  <node pkg="teleop_twist_joy" type="teleop_node" name="teleop_twist_joy">
    <remap from="cmd_vel" to="/oscar_ackermann_controller/cmd_vel" />
  </node>

</launch>
